openapi-generator/templates/go/README.mustache

```diff
@@ -41,6 +41,13 @@ func main() {
 }
 ```
 
+{{>../shared/datacenters}}
+### Specifying US Datacenter
+
+```
```

Comment:
It's not a big issue, but I would always start with a sentence rather than having only a snipped of code.

e.g.
```
Set the base path as:
```
or
```
Set the US datacenter as:
```
Similar with others.